src/mappings/technical_mapping_F03.rml.ttl

```diff
@@ -93,8 +94,8 @@ tedm:OrganisationChannel a rr:TriplesMap ;
 tedm:Organisation a rr:TripleMap ;
     rr:subjectMap
         [
-            rr:template "http://data.europa.eu/a4g/resource/Organisation/{replace(replace(/TED_EXPORT/CODED_DATA_SECTION/NOTICE_DATA/NO_DOC_OJS, ' ', '-' ), '/' , '-')}/{unparsed-text('https://digest-api.ted-data.eu/api/v1/hashing/fn/uuid/' || encode-for-uri(path()) || '?response_type=raw')}" ;
-            rr:class epo:Organisation
+            rr:template "http://www.w3.org/ns/org/Organisation/{replace(replace(/TED_EXPORT/CODED_DATA_SECTION/NOTICE_DATA/NO_DOC_OJS, ' ', '-' ), '/' , '-')}/{unparsed-text('https://digest-api.ted-data.eu/api/v1/hashing/fn/uuid/' || encode-for-uri(path()) || '?response_type=raw')}" ;
```

Comment:
Please double check this change. Seems a bit strange to use this org namespace for the instances that we create. Shouldn't it all stay in the "epd" namespace?